test/importexport.test.js

```diff
@@ -523,3 +527,29 @@ test(
   },
   testUtil.timeout.medium()
 )
+
+test(
+  'Import a ZAP file with three custom xml packages and export again, make sure order of package remains the same after export',
+  async () => {
+    sid = await querySession.createBlankSession(db)
+    await util.ensurePackagesAndPopulateSessionOptions(
+      templateContext.db,
+      sid,
+      {
+        zcl: env.builtinSilabsZclMetafile(),
+        template: env.builtinTemplateMetafile()
+      },
+      null,
+      [templatePkgId]
+    )
+    await importJs.importDataFromFile(db, threeCustomXml, {
+      sessionId: sid
+    })
+    let state = await exportJs.createStateFromDatabase(db, sid)
+
+    expect(state.package[2].path).toBe('matter-custom.xml')
```

Comment:
package[0] and package[1]?
Worth adding a multi-protocol test here where there are 2 templates.json and 2 zcl.json